Add changelog generator (#27)
kyma-project/bundles: docs/releasing.md

    ```

4. After Travis CI jobs finish, go to the **releases** tab. There you can find a new `0.6.1` release.

## Changelog functionality

> Every creation of a new release generates two changelog files, one assign to specific release and one reffering to all bundles repository.

Reviewer: ```suggestion
Every time you create a new release, two changelog files are generated. One refers to a specific release and the other one to all `bundles` repository.
```

> Changelog files contain list of all pull requests beetwen releases with labels defined as default for [lerna-changelog](https://github.com/lerna/lerna-changelog#usage). 

Reviewer: ```suggestion
Each changelog file contains a list of all pull requests that appeared between the releases, with labels defined as `default` for the [lerna-changelog](https://github.com/lerna/lerna-changelog#usage). 
```

> You can define your own labels in `scripts/package.json` file according to [lerna-changelog configuration](https://github.com/lerna/lerna-changelog#configuration).

Reviewer: ```suggestion
You can define your own labels in the `scripts/package.json` file according to the lerna-changelog [configuration](https://github.com/lerna/lerna-changelog#configuration).
```

> >**NOTE:** If you work on your fork change `scripts/package.json` configuration file setting parameters indicating on repository: `repository.url` and `changelog.repo`

Reviewer: ```suggestion
>**NOTE:** If you work on your fork, change the `scripts/package.json` configuration file and set the **repository.url** and **changelog.repo** parameters that point to the repository.
```
